examples/bridge-app/linux/Device.cpp

```diff
@@ -110,6 +110,25 @@ void DeviceOnOff::SetOnOff(bool aOn)
     }
 }
 
+void DeviceOnOff::MQTTPublisher(mqtt::async_client& client, const std::string& topic, const std::string& payload) {
```

Comment:
The function performs an action, the name should be a verb and not a noun.
```suggestion
void DeviceOnOff::MQTTPublish(mqtt::async_client& client, const std::string& topic, const std::string& payload) {
```